doc/src/explanation/gotchas.rst

```diff
@@ -332,11 +332,12 @@ automatically, but there is one gotcha you should be aware of.  If you
 do ``<number>/<number>`` inside of a SymPy expression, Python will
 evaluate the two numbers before SymPy has a chance to get
 to them.  The solution is to :func:`~.sympify` one of the numbers, or use
-:obj:`~.Rational`.
+:obj:`~.Rational` (or alternatively Python's `Fraction
```

Comment:
```suggestion
:obj:`~.Rational` (or Python's `Fraction
```

Reply:
While this is true I don't think this is going to be the easiest solution for most people. If you are going to construct a class, you should just construct the SymPy class.

Reply:
While using `sympy.Rational(fractions.Fraction(x, y))` is an overkill and I don't expect anyone to do so (unless that `Fraction` is a result of another calculation), I think it's good to make the users aware that Python also has a rational class with unlimited precision, not just `float`.